include/swift/SILOptimizer/PassManager/Passes.def

```diff
@@ -179,6 +179,12 @@ PASS(PredictableMemoryOptimizations, "predictable-memopt",
      "Predictable early memory optimizations")
 PASS(ReleaseDevirtualizer, "release-devirtualizer",
      "Devirtualize release-instructions")
+PASS(RetainSinking, "retain-sinking",
```

Comment:
We freeze epilogue releases (epilogue release matcher only finds the release in the return block currently) in till the late stage of the pipeline so we need a release hoisting and a late release hoisting. 

I experimented to hoist release first and run the first ARC pass and then sink the retains before running the second ARC pass in addSSAPasses gives the best performance.